src/Nordea/Components/FlatLink.elm

```diff
@@ -103,6 +109,47 @@ view attributes children (FlatLink config) =
         children
 
 
+inline : String -> String -> Html msg
+inline url text =
+    view [ href url ]
+        [ Html.text text ]
+        (FlatLink
+            { styles =
+                [ Css.display inlineFlex
+                , Css.fontWeight Css.inherit
+                , Themes.color Themes.PrimaryColor Colors.blueNordea
+                , hover [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
+                , focus [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
+                , visited [ Themes.color Themes.PrimaryColorLight Colors.purple ]
+                ]
+            , isDisabled = Just False
+            , variant = Default
+            }
+        )
+
+
+outer : String -> String -> Html msg
+outer url text =
+    view [ href url ]
+        [ Html.text text, Icons.rightIcon (Icons.info []) ]
+        (FlatLink
+            { styles =
+                [ Css.fontWeight Css.inherit
+                , Themes.color Themes.PrimaryColor Colors.blueNordea
+                , textDecoration none
+                , hover
+                    [ Themes.color Themes.PrimaryColorLight Colors.blueDeep
+                    , textDecoration underline
+                    ]
+                , focus [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
+                , visited [ Themes.color Themes.PrimaryColorLight Colors.purple ]
+                ]
+            , isDisabled = Just False
```

Comment:
kinda out of scope, but should this Maybe Bool just be Bool with default False instead of Nothing?

Reply:
lgtm